values.yaml

```diff
@@ -149,8 +149,8 @@ persistence:
       #realm: core.windows.net
     gcs:
       bucket: bucketname
-      # TODO: support the keyfile of gcs
-      #keyfile: /path/to/keyfile
+      # The base64 encoded json file which contains the key
+      keyfile: base64-encoded-json-key-file
```

Comment:
This is the key data rather than the key file, right?
Is it a convention to put the data in attribute `keyfile` when configuring gcs?

Reply:
Yes, it is the content of the key file. And the chart of chartmusuem uses the same way to support gcs.

Reply:
So is it appropriate to call it `keyfile` here?  maybe I was misled by the removed comment, but it may make user think he should put the file path here.